widgetry/src/text.rs

```diff
@@ -422,12 +418,13 @@ fn render_line(spans: Vec<TextSpan>, tolerance: f32, assets: &Assets) -> GeomBat
 
     write!(
         &mut svg,
-        r##"<text x="0" y="0" font-size="{}" font-family="{}" {}>"##,
+        r##"<text x="0" y="0" xml:space="preserve" font-size="{}" font-family="{}" {}>"##,
```

Comment:
Even if we ditch the rest of this PR, we might want to keep this part. I was surprised that whitespace was being dropped.

(though maybe since trailing whitespace is dropped due to https://github.com/RazrFalcon/resvg/issues/317 it's at least consistent as it is now)

Reply:
Agreed, let's keep this. I didn't know about this attribute. I'd love to use this instead of "- " in places like `info/intersection.rs` and `info/trip.rs`.

Reply:
Done here: https://github.com/dabreegster/abstreet/pull/304